velox/docs/functions/spark/aggregate.rst

```diff
@@ -7,6 +7,15 @@ Aggregate functions operate on a set of values to compute a single result.
 General Aggregate Functions
 ---------------------------
 
+.. spark:function:: avg(x) -> double|decimal
+
+    Returns the average (arithmetic mean) of all non-null input values.
+    When x is of type DECIMAL, the result type is DECIMAL.
+    For all other input types, the result type is DOUBLE.
+    Intermediate results are (sum, count) pairs represented as row(double, bigint).
```

Comment:
perhaps, clarify the intermediate type for decimal inputs.

Reply:
Added decimal part.